configure.ac

```diff
@@ -159,7 +159,7 @@ AC_CHECK_SIZEOF(bfd_vma, [], [#include <bfd.h>])
 # Checks for library functions.
 AC_FUNC_MALLOC
 AC_FUNC_MEMCMP
-AC_CHECK_FUNCS([bzero memset strstr strtoul])
+AC_CHECK_FUNCS([bzero memset strstr strtoul malloc])
```

Comment:
Could you expand on the circumstances where you had a failure related to rpl_malloc, and why this change is the best fix for the problem? My understanding is that configure creates rpl_malloc when it cannot confirm that malloc(0) returns a non-null value. But Linux malloc works that way, so in my experience this failure is more often caused by some other problem with your C compiler that prevents the test program from compiling or running correctly.

Adding "malloc" to AC_CHECK_FUNCS checks whether "malloc" exists, but presumably malloc does exist on your system. If adding malloc to AC_CHECK_FUNCS fixes your problem only because it overrides "AC_FUNC_MALLOC", removing the latter seems like the change to consider.

Thanks,

 -- Stephen